sleap_nn/config/data_config.py

```
@@ -0,0 +1,13 @@
+@attr.s(auto_attribs=True)
```

Ruff

Undefined name `attr`

(F821)